include/teraranger/teraranger_duo.h

```diff
@@ -0,0 +1,98 @@
+/****************************************************************************
```

Comment:
I would remove those header strings, this info should be contained in package.xml (and maybe in a licence file)

Reply:
done